vendor/github.com/evanphx/json-patch/patch.go

```diff
@@ -568,29 +568,6 @@ func (p Patch) replace(doc *container, op Operation) error {
 		return errors.Wrapf(err, "replace operation failed to decode path")
 	}
 
-	if path == "" {
```

Comment:
it looks like this lost this fix: https://github.com/evanphx/json-patch/commit/50fdc0b4c9ab36933e6a0c9288fef3ac5df2b907